app/components/DeleteButton/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+*
+* DeleteButton
+*
+*/
+
+import React from 'react';
+
+import styles from './styles.css';
+
+function DeleteButton(props) {
+  return (
+    <button
+      className={styles.deleteButton}
+      onClick={() => {
+        const response = window.confirm(props.confirmText); // eslint-disable-line no-alert
```

Comment:
Alright. A general note on ESlint-disabling: 
It's nice to add some comment before an ESlint-disabler that explains to others why it's disabled for that line. In other words, there should be a good reaso, i.e. framework X does not work unless it's done this way.  ESlint-rules [are there for a reason](http://eslint.org/docs/rules/no-alert), so if you want to circumvent them it should be explained. 

Regarding this rule: Rules like that one, could be turned off if you actually want to be able to do alerts, but that is a UX-decision whcih should be cleared with the team and the PO. (Also see the comment below).